data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3483 @@
+			`	The same Chief is now moving in your general direction. He seems to be lost in his thought and is mumbling on, something about, "We do not get alien visitors often here. There has to be a reason for your presence..." When he speaks again, directly to you, it is with more focus, "Sorry, I was just thinking out loud! Well you are here anyway so maybe you could help us?"`
+			`	"We have friends outside of our space, civilians and others. You see, we would do anything to help a friend, and a few Hai want to join us. I can only hope you are willing to help us reunite with our brothers and sisters from Hai space," he says, apparently a bit desperate, considering he knows very little if anything at all about you.`
+			choice
+				`	"Of course, what do you need me to do?"`
+					goto action
+				`	"What do you mean, brother and sisters from Hai space? Are you not different?"`
+					goto brethren
+				`	"Why do you need me to do it?"`
+				`	"This seems like a bad idea, ask someone else."`
+					decline
+			
+			label whyme
+			`	"Well normally they would be able to pay to come here, but the price is pretty high; too high. The Hai make it difficult for us to go anywhere; as you have probably noticed they have a fleet at the edge of our system. And not many Hai captains are willing to bring them here, or at least they ask for a big fee, because of the obvious risks involved, like getting killed in all that cross fire. Yet you somehow made your way here, probably also under fire from our own people, but now that we are allied, you hopefully should not have any issues getting out of here. So, are you willing to help us?"`
+			choice
+				`	"Of course, what do you need me to do?"`
+					goto action
+				`	"This seems like a bad idea, ask someone else."`
+					decline
+			
+			label brethren
+			`	"Then you have heard about the stories of the Drak modifying our people..." He seems a bit sad. "I do not believe in such things, what makes a Hai -or any other species- “True” is in his heart. It will take him wherever he wants to go." He pauses a bit, thinking, and when he speaks again it is with deep conviction. "Even if we war with the Elders it does not mean we hate other Hais or them us! We do it only because we must, in order to keep living our way. We can still communicate with our people, and eventually they will all be freed from the limits of the Elders."`
```

Comment:
```suggestion
			`	"Then you have heard about the stories of the Drak modifying our people..." He seems a bit sad. "I do not believe in such things, what makes a Hai - or any other species - “true” is in his heart. It will take him wherever he wants to go." He pauses a bit, thinking, and when he speaks again it is with deep conviction. "Even if we war with the Elders it does not mean we hate other Hai, or they hate us! We do it only because we must, in order to keep living our way. We can still communicate with our people, and eventually they will all be freed from the limits of the Elders."`
```